frontend/packages/dev-console/locales/en/devconsole.json

```diff
@@ -643,7 +643,8 @@
   "Average Container bandwidth by Pod: transmitted": "Average Container bandwidth by Pod: transmitted",
   "Successfully updated the project access.": "Successfully updated the project access.",
   "Project access": "Project access",
-  "Project access allows you to add or remove a user's access to the project. More advanced management of role-based access control appear in <1>Roles</1> and <4>Role Bindings</4>. For more information, see the <7>role-based access control documentation</7>.": "Project access allows you to add or remove a user's access to the project. More advanced management of role-based access control appear in <1>Roles</1> and <4>Role Bindings</4>. For more information, see the <7>role-based access control documentation</7>.",
+  "Project access allows you to add or remove a user's access to the project. More advanced management of role-based access control appear in <1>Roles</1> and <4>Role Bindings</4>.": "Project access allows you to add or remove a user's access to the project. More advanced management of role-based access control appear in <1>Roles</1> and <4>Role Bindings</4>.",
+  " For more information, see the <3>role-based access control documentation</3>.": " For more information, see the <3>role-based access control documentation</3>.",
```

Comment:
Do we need the extra space at the beginning ? I see that we are adding it manually.